plugin/log/log.go

```diff
@@ -54,7 +54,7 @@ func (l Logger) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg)
 
 		tpe, _ := response.Typify(rrw.Msg, time.Now().UTC())
 		class := response.Classify(tpe)
-		if rule.Class == response.All || rule.Class == class {
+		if rule.Class[response.All] || rule.Class[class] {
```

Comment:
Is `rule.Class[class]` always in the map? If not this crashes runtime

Reply:
Yes, it is, even if we don't set up a class in config, the default "all" will be added, hence, we shouldn't have an empty rule.Class